applications/crossbar/priv/couchdb/schemas/devices.json

```diff
@@ -406,6 +406,11 @@
             },
             "type": "object"
         },
+        "rtcp_mux": {
+            "default": true,
```

Comment:
@sergey-safarov please remove the default if its the same value as expected, ie, no need to tell freeswitch to set a value that it already have.